proto/dropbox/dropbox.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package dropbox;
+
+service Dropbox {
+  rpc GetApproverInfo(GetApproverInfoRequest) returns(GetApproverInfoResponse) {
+  }
+  rpc CanApproveToUser(CanApproveToUserRequest)
+      returns(CanApproveToUserResponse) {}
+  rpc CreateRequest(CreateRequestRequest) returns(CreateRequestResponse) {}
+  rpc HasTransfer(HasTransferRequest) returns(HasTransferResponse) {}
+  rpc GetTransfersInfo(GetTransfersInfoRequest)
+      returns(GetTransfersInfoResponse) {}
+}
+
+message Unit {
+  string name = 1;
+  repeated string approvers = 2;
+}
+
+message ApproverInfo {
+  string userId = 1;
+  bool isAdmin = 2;
+  bool isApprover = 3;
+  bool isBlocked = 4;
+  Unit unit = 5;
```

Comment:
Can there be an approvers array inside this unit that is inside an `approver` object? 🤔 
If so, it's hard for me to understand why.
If not, you should just put a `unitName` field.

Reply:
This is what i get from dropbox.

Reply:
OK. How do you use it afterward? Do you filter the approvers you display to the users in the UI?